### PR TITLE
Edit optimize-writes.md

### PR DESCRIPTION
**Pull request by pierwill**

### CODE REVIEW

File: content/v2.0/write-data/best-practices/optimize-writes.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ The following tools write to InfluxDB and employ write optimizations by default:
 
 ## Batch writes
 
-Write data in batches to Minimize network overhead when writing data to InfluxDB.
+Write data in batches to minimize network overhead when writing data to InfluxDB.
 
 {{% note %}}
 The optimal batch size is 5000 lines of line protocol.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/write-data/best-practices/optimize-writes.md]: The SEs recommend batches between 5k and 10k **lines**, but this recommendation should be based on points, not lines. They are currently in the process of reviewing their recommendations based on actual ingest metrics on Cloud 1 and Cloud 2.

File: content/v2.0/write-data/best-practices/optimize-writes.md [hunk 2]
```diff
@@ -44,10 +44,9 @@ measurement,tagA=i,tagB=think,tagC=therefore,tagD=i,tagE=am fieldKey=fieldValue
 
 ## Use the coarsest time precision possible
 
-InfluxDB lets you write data in nanosecond precision, however if data isn't
-collected in nanoseconds, there is no need to write at that precision.
-Using the coarsest precision possible for timestamps can result in significant
-compression improvements.
+By default, InfluxDB writes data in nanosecond precision.
+However if your data isn't collected in nanoseconds, there is no need to write at that precision.
+Using the coarsest precision possible for timestamps can result in significant performance improvements.
```
Comment (sanderson) [inline content/v2.0/write-data/best-practices/optimize-writes.md]: I know I write this originally 😄, but what about:

> For better performance, use the coarsest precision possible for timestamps.

File: content/v2.0/write-data/best-practices/optimize-writes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pierwill) [reply content/v2.0/write-data/best-practices/optimize-writes.md]: I'll leave as-is for now. We'll want to circle back to this—

[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/write-data/best-practices/optimize-writes.md]: 👍 For sure

**Pull request merged**